xbmc/video/guilib/VideoActionProcessorBase.h

```diff
@@ -26,15 +26,18 @@ class CVideoActionProcessorBase
   bool ProcessAction(Action action);
 
   bool UserCancelled() const { return m_userCancelled; }
+  void SetUserCancelled(bool set) { m_userCancelled = set; }
+
+  std::shared_ptr<CFileItem> GetItem() const { return m_item; }
```

Comment:
I think for const correctness it should be this:
```suggestion
  std::shared_ptr<CFileItem> GetItem() { return m_item; }
  std::shared_ptr<const CFileItem> GetItem() const { return m_item; }
```
This won't compile because of stuff in VideoPlayActionProcessor.cpp which is IMO the correct behavior, e.g. an "active" function like `SetResumeData()` can not be `const`.